### PR TITLE
fake spelling errors

### PR DESCRIPTION
**Pull request by joebumbulis**
Please check for my fake spell errors in the H1

### CODE REVIEW

**Review by Spica8080 — CHANGES_REQUESTED** [review]
i see it!  👍

File: index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
   </head>
   <body>
     <img src="assignments/images/bike logo clipart.png" alt="">
-    <h1>Barton Creek Cycling</h1>
+    <h1>Baron Crek Ccling</h1>
```
Comment (ghost) [inline index.html:10]: Spelling error "Creek Cycling"

**Review by ghost — COMMENTED** [review]
There is a spelling error "Creek Cycling".

File: index.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
   </head>
   <body>
     <img src="assignments/images/bike logo clipart.png" alt="">
-    <h1>Barton Creek Cycling</h1>
```
Comment (joebumbulis) [inline index.html:10]: Keep this spelling.

**Pull request merged**